test/file.ts

```diff
@@ -3853,6 +3865,129 @@ describe('File', () => {
 
   describe('save', () => {
     const DATA = 'Data!';
+    const BUFFER_DATA = Buffer.from(DATA, 'utf8');
+
+    class DelayedStreamNoError extends Transform {
+      _transform(chunk: any, _encoding: string, done: Function) {
```

Comment:
might b e worth giving chunk a type, I believe it will be a `Buffer` or `string`.